membership-common/src/main/scala/com/gu/memsub/subsv2/services/CatalogService.scala

```diff
@@ -42,125 +34,23 @@ object FetchCatalog {
 
 }
 
-class CatalogService[M[_]: Monad](productIds: ProductIds, fetchCatalog: M[String \/ JsValue], unsafeGet: M[Catalog] => Catalog, stage: String) {
```

Comment:
🔪  🔪  🔪 don't need to use the CatPlanReads to construct the high level model as it's not used!